app/controllers/development_controller.rb

```diff
@@ -1,4 +1,6 @@
 class DevelopmentController < ApplicationController
+  include Slimmer::Template
+  slimmer_template "gem_layout"
```

Comment:
I think that these two lines need to be in `application_controller.rb`, rather than in `developerment_controller.rb`:

```ruby
  include Slimmer::Template
  slimmer_template "gem_layout"
```

Reply:
It seems that I've lost some code after rebasing as https://github.com/alphagov/government-frontend/pull/2235/files#diff-8f720467cd4cfd6b9a611598b9491f0ffa8582cd2d260b733a36e16c0179bfd0L62 got merged and removed slimmer_template declarations where `gem_layout` was used. Will update.